### ISSUE TITLE
LOGNOTICE deprecated in KODI

### ISSUE THREAD

**Issue by XayOn** [jellyfin/jellyfin-kodi#396]
As per six days ago, kodi removed LOGNOTICE and LOGSEVERE See (https://github.com/xbmc/xbmc/pull/18346)

It's currently being used at https://github.com/jellyfin/jellyfin-kodi/blob/d13114f3abe082cfd35830f8f7798c49e6a81513/jellyfin_kodi/helper/loghandler.py#L64

According to https://forum.kodi.tv/showthread.php?tid=353818 LOGINFO should be used

**Comment by oddstr13** [comment]
Thanks!
Looks like this will require a little bit more than just replacing the loglevel, as `LOGINFO` doesn't output to log by default in Kodi 18 according to the linked info.
Should be easy enough to set a module level variable based on the Kodi version though.

**Comment by XayOn** [comment]
@oddstr13 I think it could be simplified it using getattr.

`getattr(xbmc, 'LOGNOTICE', xbmc.LOGINFO)`

I saw that message but couldn't confirm it as I've exclusively been on KODI Matrix for a while. 

Pros of gettattr approach:
- No need to check for kodi version, it'll be specific to that use-case
Cons of getattr approach:
- The logic will be a bit darker (wich could be clarified by a comment)

Pros of version approach:
- The logic will be more readable
Cons of version approach:
- Requires extracting kodi version

I normally would be against the extracting kodi version approach, as I always go with the "don't be afraid of future change" philosofy, implying that, if a future another kodi-version specific change is required, this could be refactored to the kodi version approach because kodi-version-specific logic would be used in both places. But the decision is yours. 

I've added a 396-b branch and made another pull request to this issue, so both approaches are covered
